src/components/ReportActionItem/IOUQuote.js

```diff
@@ -30,27 +40,45 @@ const IOUQuote = ({
     shouldShowViewDetailsLink,
     onViewDetailsPressed,
     translate,
-}) => (
-    <View style={[styles.chatItemMessage]}>
-        {_.map(action.message, (fragment, index) => (
-            <View key={`iouQuote-${action.sequenceNumber}-${index}`}>
+    numberFormat,
+    currentUserFirstName,
+    participantFirstName,
+    sessionEmail,
+}) => {
+    const {
+        type: messageType, amount, currency, comment, paymentType,
+    } = action.originalMessage;
```

Comment:
I'm confused why we switched to using `originalMessage` here? Is there some explanation?

Reply:
`originalMessage` has the individual properties that can be accessed.  `message` is just plain text with the full statement in English.

Reply:
Oh hmm that is pretty weird. I'm not sure then. Maybe @Julesssss knows for sure and can advise better here.

Reply:
Sample `action` object for your reference.

![image](https://user-images.githubusercontent.com/3069065/130684887-3458d5ed-9b06-4ffd-820e-a5eadc36b6c3.png)

Reply:
@Julesssss Can you help a bit here as well?

Reply:
I think the change makes sense.

The `originalMessage` name is a bit odd, but it is an artifact coming from the server (we had to differentiate between 'literal message' and 'the data which was used to generate the original message')